### PR TITLE
feat(icons): Add IconPlusRound and IconCaretRound for collections

### PR DESCRIPTION
**Pull request by shahzadaziz**
Screen Shot 2019-11-15 at 2 42 07 PM

Screen Shot 2019-11-15 at 2 42 26 PM

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @shahzadaziz has signed the CLA. Thanks for the pull request!

File: src/icons/general/IconPlusRound.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+// @flow
+import * as React from 'react';
+
+import AccessibleSVG from '../accessible-svg';
+import { bdlGray80 } from '../../styles/variables';
+
+import type { Icon } from '../flowTypes';
+
+const IconPlusCircle = ({ className = '', color = bdlGray80, height = 16, title, width = 16 }: Icon) => (
```
Comment (priyajeet) [inline src/icons/general/IconPlusRound.js]: File name is IconPlusRound

File: src/icons/general/IconPlusRound.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    <AccessibleSVG
+        className={`bdl-IconPlusRound ${className}`}
+        height={height}
+        title={title}
+        viewBox="0 0 16 16"
+        width={width}
+    >
+        <g className="fill-color" fill={color}>
```
Comment (priyajeet) [inline src/icons/general/IconPlusRound.js]: You can just have classname and fill directly on the path

File: src/icons/general/IconCaretRound.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+const IconCaretRound = ({ className = '', color = bdlGray80, height = 16, title, width = 16 }: Icon) => (
+    <AccessibleSVG
+        className={`bdl-IconCaretRound ${className}`}
+        height={height}
+        title={title}
+        viewBox="0 0 16 16"
+        width={width}
+    >
+        <g className="fill-color" fill={color}>
```
Comment (priyajeet) [inline src/icons/general/IconCaretRound.js]: same as other comment

**Comment by priyajeet** [comment]
screenshots seem busted

**Comment by shahzadaziz** [comment]
@priyajeet addressed your comments.

**Pull request merged**